src/functionalTest/resources/features/F-004/F-000.feature

```diff
@@ -0,0 +1,122 @@
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a non-existing classification]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-047
+  Scenario: must get an error response for a malformed classification
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed classification]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-048
+  Scenario: must get an error response for a non-existing roles
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a non-existing roles]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-049
+  Scenario: must get an error response for a malformed roles
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed roles]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-050
+  Scenario: must get an error response for a non-existing roles
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a non-existing roles]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-051
+  Scenario: must get an error response for a malformed roles
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed roles]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-052
```

Comment:
We can't check the user-id even if the user is not having create permission on the case, he would be able to upload document with 10 minute TTL that will delete the document because ccd side create/update event will fail due to insufficient permission.

Reply:
Ignore this test and put the comment that this test will be done manually